modules/products/bitbucket/variables.tf

```diff
@@ -115,44 +115,71 @@ variable "shared_home_size" {
   default     = "10Gi"
 }
 
-# If an external elasticsearch is not provided, Bitbucket will provision an elasticsearch cluster in k8s
-variable "elasticsearch_endpoint" {
-  description = "The external elasticsearch endpoint to be use by Bitbucket."
+variable "deploy_opensearch" {
+  description = "Install OpenSearch sub-chart with Bitbucket Helm chart"
+  type        = bool
+  default     = true
+}
+
+# If an external opensearch is not provided, Bitbucket will provision an opensearch cluster in k8s
+variable "opensearch_endpoint" {
+  description = "The external opensearch endpoint to be use by Bitbucket."
```

Comment:
```suggestion
  description = "The external OpenSearch endpoint to be used by Bitbucket."
```

Reply:
There are other `opensearch` references. I leave that up to you if you want to make them consistent or not. Right now the casing is a bit weird but nothing really important.